drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -253,6 +254,23 @@ def _learner_raw_score(self):
         correct_count, total_count = self._get_item_stats()
         return correct_count / float(total_count)
 
+    @staticmethod
+    def _get_statici18n_js_url():
```

Comment:
This method has many nesting depth and cases, it'd be much readable if we flatten it more. [Statement nesting depth](https://help.semmle.com/wiki/display/PYTHON/Statement+nesting+depth)